doc/tutorials/mount.md

```diff
@@ -22,6 +22,9 @@ We mount the lookup table with the following command:
 2. `system/hosts` is the path it should have in the key database, also known as **mountpoint**
 3. `hosts` is the _storage plugin_ that can read and write this configuration format.
 
+> Consider using mount with the option `--with-recommends`, which loads all plugins recommended by the _hosts_ plugin.
+> You can see the recommendet plugins of _hosts_ if you look at the output of `kdb info hosts`. Hosts recommends the _glob_ and _network_ plugins.
```

Comment:
error plugin is now also recommended (to inject error by glob for keys not within ipv4 or ipv6)